.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Feature request
+about: Ask for a new feature to be added (module, program, etc.)
+title: '<title>'
```

Comment:
I don't love how this auto-populates the field:

![image](https://user-images.githubusercontent.com/16734772/125524384-200d1502-888f-4f43-adfc-d2fc23fd1ca7.png)

Reply:
Fixed